cmd/generator/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"os"
+)
+
+func main() {
+	argsWithProg := os.Args
+
+	if len(argsWithProg) != 2 {
+		panic("Wrong amount of arguments!")
+	}
+
+	if len(argsWithProg[1]) == 0 {
+		panic("Given regex is empty!")
+	}
+
+	regex := argsWithProg[1]
+
+	_, err := os.Stdout.WriteString((generateCode(regex) + "\n"))
+	if err != nil {
+		panic("Error writing to stdout")
+	}
+}
+
+func generateCode(regex string) string {
+	afterLastBrace, beforeCurrentBrace, indentations, result := regex, "", "", "rex.New(\n"
```

Comment:
```suggestion
	var (
		afterLastBrace     = regex
		beforeCurrentBrace = ""
		indentations       = ""
		result             = "rex.New(\n"
	)
```

Why?

In `afterLastBrace, beforeCurrentBrace, indentations, result := regex, "", "", "rex.New(\n"` is difficult to see what is the value of `indentations`.

Reply:
Ok, I will change it :)